README.md

```diff
@@ -50,7 +50,8 @@ See the [OpenTracing website](http://opentracing.io/) for general information on
 The project examples are run by . Run:
```

Comment:
Is there something missing at the end of this sentence? Should it read, "The project examples are run by `webdev_proxy`."

Reply:
yep, I'll add it